extension/lib/hooks/getRootCategories-after-addSmartCategories.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = async (context, input) => {
+  if (!context.config.smartCategories) {
+    return input
```

Comment:
It is nicer if we don't use input as output

Reply:
done